cmd/root.go

```diff
@@ -227,12 +227,12 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&debugSkipFiles, "debug-skip-files", "", "Used when debugging, to tell AzCopy to cancel the job midway. List of relative paths to skip in the STE.")
 
 	// reserved for partner teams
-	rootCmd.PersistentFlags().MarkHidden("cancel-from-stdin")
+	_ = rootCmd.PersistentFlags().MarkHidden("cancel-from-stdin")
```

Comment:
ditto: re nolinting this as the code quality increase is negligible/unneccessary

Reply:
+1

Reply:
I don't anticipate adding too many more of these so I'd prefer to leave it here instead of me fishing for all these types of fixes that I already made.